packages/typescript-resolver-files/src/utils/index.ts

```diff
@@ -5,10 +5,18 @@ export * from './isNativeNamedType';
 export * from './isRootObjectType';
 export * from './printImportLine';
 
+function slash(path:string) => string {
+  const isExtendedLengthPath: boolean = /^\\\\\?\\/.test(path);
+
+  if(isExtendedLengthPath) return;
+
+  return path.replace(/\\/g,'/');
+}
```

Comment:
Sorry @JustusFluegel , this doesn't seem to be valid TypeScript.

It'll fail when you try to compile using this command: `yarn nx build typescript-resolver-files`

Reply:
yea i am stupid i used => for type annotations, that is rust style i guess, will fix after dinner.

Reply:
should be fixed